parsers/US_PJM.py

```diff
@@ -125,39 +131,37 @@ def fetch_production(
         "datetime_beginning_ept": target_datetime.strftime("%Y-%m-%dT%H:00:00.0000000"),
     }
     resp_data = fetch_api_data(kind="gen_by_fuel", params=params, session=session)
-    data = pd.DataFrame(resp_data.get("items", []))
-    if not data.empty:
-        data["datetime_beginning_ept"] = pd.to_datetime(data["datetime_beginning_ept"])
-        data = data.set_index("datetime_beginning_ept")
-        data["fuel_type"] = data["fuel_type"].map(FUEL_MAPPING)
-
-        all_data_points = []
-        for dt in data.index.unique():
-            production = {}
-            storage = {}
-            data_dt = data.loc[data.index == dt]
-            for i in range(len(data_dt)):
-                row = data_dt.iloc[i]
-                if row["fuel_type"] == "battery":
-                    storage["battery"] = row.get("mw")
-                else:
-                    mode = row["fuel_type"]
-                    production[mode] = row.get("mw")
```

Comment:
This previously caused us to loose data, in the snapshot it was just 1 MW per hour but still.